calcom/cal.com
# enhance Timezone Component with Box Shadow

## What does this PR do?

i have added boxshadow to timezone options container

Fixes #8159 



**Environment**:  Production

## Type of change



- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?

1. Verify that the TimezoneSelect component renders without any errors.
2. Open the dropdown menu of the TimezoneSelect component.
3 .Check that the container holding the options has the desired box shadow applied.
4. Select different options and ensure that the box shadow remains consistent.

before
![beforecalcom](https://user-images.githubusercontent.com/110285294/235365011-1acbfccb-364b-4563-b765-0c9f905d3915.png)

after
![calcom](https://user-images.githubusercontent.com/110285294/235364774-cb72e003-d788-4f8d-ab3f-1c89fddace2a.jpg)
